frontend/src/service/RequestService.ts

```diff
@@ -33,6 +34,21 @@ export const createBagPlace = (newMarker: BagPlaceCreationDTO, token?: string) =
         })
 }
 
-export const getBagPlaces = () =>
-    axios.get("/api/bagplaces")
-        .then(response => response.data)
\ No newline at end of file
+
+export const addAppointment = (newAppointment: AppointmentDTO, token?: string) => {
```

Comment:
Is this supposed to be called by an unauthorized user? If not make token required and remove the ternary

Reply:
The function should be callable by users who aren't logged in, but should immediately end in an error with an alert to log in before trying to safe any appointments. The alert part is missing because the function is not fully written atm.
Previously, i had some problems after making the token required in my 'createBagPlace' function, and because of that i copied my code of 'createBagPlace' to 'addAppointment